osidb/models.py

```diff
@@ -940,6 +940,21 @@ def _validate_nist_rh_cvss_feedback_loop(self):
                 f"an explanation comment for RH CVSSv3 score.",
             )
 
+    def _validate_cvss_scores_and_nist_cvss_validation(self):
```

Comment:
Should I add the same validation into the `FlawCVSS` class?

Reply:
Good question. I think that this validation could be typically triggered by assigning a flag to a flaw when there should be none and not by assigning a CVSS to it so I think this direction is enough. The other question is getting rid of this error as if one simply adds a CVSS it does not trigger `flaw.save` so the alert would stay there. I would suggest to make the alert blocking. There is no reason why we should allow an invalid flag. Then there will also be no dangling alert.